common/api/summary/Ben.exports.csv

```diff
@@ -0,0 +1,823 @@
+sep=;
```

Comment:
what is this file?

Reply:
Not sure what causes api-extractor to sometimes produce username.exports.csv. Deleted.